src/pages/teaching-tech.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+
+import Layout from '../components/Layout'
+import ConstrainedWidth from '../components/Layout/ConstrainedWidth'
+import Heading from '../components/Heading'
+import Text from '../components/Text'
+import VerticalSpacing from '../components/VerticalSpacing'
+import { ExternalLink } from '../components/Link'
+import PageMeta from '../components/PageMeta'
+import Centered from '../components/Centered'
+
+const TeachingTech = () => (
+  <Layout>
+    <PageMeta
+      title="Teaching Tech"
+      description="Want to get better at helping developers learn? Join our accredited course to improve
+              your teaching skills"
```

Comment:
Is there a neater way to break this line? The highlighting in github indicates there is something odd going on.